packages/teleport/src/Databases/ConnectDialog/ConnectDialog.test.tsx

```diff
@@ -39,14 +39,23 @@ test('correct connect command generated for mysql db', () => {
 
 test('correct tsh login command generated with local authType', () => {
   render(<ConnectDialog {...props} />);
-  const output = 'tsh login --proxy=localhost:443 --auth=local --user=yassine';
+  const output =
+    'tsh login --proxy=localhost:443 --auth=local --user=yassine im-a-cluster';
 
   expect(screen.getByText(output)).toBeInTheDocument();
 });
 
 test('correct tsh login command generated with sso authType', () => {
   render(<ConnectDialog {...props} authType="sso" />);
-  const output = 'tsh login --proxy=localhost:443';
+  const output = 'tsh login --proxy=localhost:443 im-a-cluster';
+
+  expect(screen.getByText(output)).toBeInTheDocument();
+});
+
+test('correct tsh login command generated with passwordless authType', () => {
```

Comment:
Hi @hatched, I've added a test case to verify that the `tsh login` command is correct when passwordless `authType` is used. Do you have some recommendations for other test cases that could be added/modified?

Reply:
Nope this looks good 👍